Only build/install panel applet if python-gnomeapplet is available.

--- origsrc/revelation-0.4.11/acinclude.m4	2006-07-22 10:22:44.000000000 -0500
+++ src/revelation-0.4.11/acinclude.m4	2010-02-16 15:26:30.162232700 -0600
@@ -83,7 +83,6 @@ AC_DEFUN([RVL_MMAN], [
 AC_DEFUN([RVL_PYGTK], [
 	PKG_CHECK_MODULES(PYGTK, [pygtk-2.0 >= 2.8.0])
 	PKG_CHECK_MODULES(GNOME_PYTHON, [gnome-python-2.0 >= 2.10.0])
-	PKG_CHECK_MODULES(GNOME_PYTHON_EXTRAS, [gnome-python-extras-2.0 >= 2.10.0])
 
 	AC_PATH_PROG(PYGTK_CODEGEN, pygtk-codegen-2.0, no)
 
@@ -107,8 +106,10 @@ AC_DEFUN([RVL_PYTHON_MODULE], [
 		eval AS_TR_CPP(HAVE_PYMOD_$1)=yes
 	else
 		AC_MSG_RESULT(no)
+		if test x$2 == xyes; then
 		AC_MSG_ERROR(failed to find module $1)
 		exit 1
+		fi
 	fi
 ])
 
--- origsrc/revelation-0.4.11/configure.ac	2010-02-16 14:42:46.835186900 -0600
+++ src/revelation-0.4.11/configure.ac	2010-02-16 15:27:02.295070600 -0600
@@ -28,7 +28,8 @@ RVL_PYTHON_MODULE(bonobo, yes)
 RVL_PYTHON_MODULE(gconf, yes)
 RVL_PYTHON_MODULE(gtk, yes)
 RVL_PYTHON_MODULE(gnome, yes)
-RVL_PYTHON_MODULE(gnomeapplet, yes)
+RVL_PYTHON_MODULE(gnomeapplet, no)
+AM_CONDITIONAL(WITH_PANELAPPLET, test x$HAVE_PYMOD_GNOMEAPPLET = xyes)
 RVL_PYTHON_MODULE(gnomevfs, yes)
 RVL_PYTHON_MODULE(gobject, yes)
 RVL_PYTHON_MODULE(pango, yes)
--- origsrc/revelation-0.4.11/data/bonobo/Makefile.am	2010-02-16 14:42:47.993253200 -0600
+++ src/revelation-0.4.11/data/bonobo/Makefile.am	2010-02-16 15:33:00.919582700 -0600
@@ -5,8 +5,10 @@
 # $Id: Makefile.am 535 2006-06-11 18:17:22Z erikg $
 #
 
+if WITH_PANELAPPLET
 serverdir		= $(libdir)/bonobo/servers
 server_DATA		= GNOME_RevelationApplet.server
+endif
 CLEANFILES		= GNOME_RevelationApplet.server
 EXTRA_DIST		= GNOME_RevelationApplet.server.in
 
--- origsrc/revelation-0.4.11/data/gconf/Makefile.am	2006-06-12 14:05:11.000000000 -0500
+++ src/revelation-0.4.11/data/gconf/Makefile.am	2010-02-16 15:35:43.680892100 -0600
@@ -6,7 +6,10 @@
 #
 
 schemadir		= @GCONF_SCHEMA_FILE_DIR@
-dist_schema_DATA	= revelation.schemas revelation-applet.schemas
+dist_schema_DATA	= revelation.schemas
+if WITH_PANELAPPLET
+dist_schema_DATA	+= revelation-applet.schemas
+endif
 
 
 install-data-hook:
--- origsrc/revelation-0.4.11/src/Makefile.am	2010-02-16 14:42:48.036255600 -0600
+++ src/revelation-0.4.11/src/Makefile.am	2010-02-16 15:19:34.768473500 -0600
@@ -7,7 +7,9 @@
 
 SUBDIRS		= bundle lib wrap
 bin_SCRIPTS	= revelation
+if WITH_PANELAPPLET
 libexec_SCRIPTS	= revelation-applet
+endif
 EXTRA_DIST	= revelation.in revelation-applet.in
 CLEANFILES	= revelation revelation-applet
 
